mautic/mautic
# [UI] Replace header icons from FontAwesome to Remix

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Replaced to the new icon pack.
Added the user's position above their name.

---
Reordered the notification icon to improve UX.

In the matter of information distribution, we easily realize that items on the right side of the screen are more related to technical management or administration. Either resources such as categories as well as the settings tab, having them all in one side tells users we have a consistency. 

So from this, we can think it would also make sense to keep the notification panel icon on the right side of the screen, since often it is responsible for showing information as a new update available. 

The left side, on the other hand, commonly presents items that marketers use in everyday work, like contacts, campaigns as well as other necessary resources for the success of a strategy. I believe that keeping only the search icon alone in this side improves consistency, since they will use global search to find these items easily.

## Before:
![image](https://github.com/mautic/mautic/assets/116097999/9df66528-9384-4d07-8d5f-5f063388ca95)

![image](https://github.com/mautic/mautic/assets/116097999/d1f680ab-d86c-45a0-b4a7-56fa36de994a)

![image](https://github.com/mautic/mautic/assets/116097999/0d68dd4c-f26b-47e1-8358-f14cd495bca5)


## After:


https://github.com/mautic/mautic/assets/116097999/238ff04d-63ee-4397-a818-c7a93c0332fb


![image](https://github.com/mautic/mautic/assets/116097999/e2ce4f1d-6c4b-4433-aeb3-e19195bb7a0d)

![image](https://github.com/mautic/mautic/assets/116097999/4aa1d47f-3bc9-4a4f-8825-5bdcf38dbd30)

![image](https://github.com/mautic/mautic/assets/116097999/9cc4991d-2928-46e2-9bf1-6b32ac4c10a3)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Look at the navbar
